rockcraft.yaml

```diff
@@ -0,0 +1,30 @@
+name: kratos
+base: bare
+build-base: ubuntu:22.04
+version: "0.13.0"
```

Comment:
If we're going to have multiple image versions at the same time (e.g. one currently used by kratos-operator and a newer one in development / tests), should we name the rockcraft files differently to distinguish them or perhaps parametrize the version?

Reply:
You are right that the versioning needs to be worked on, but I think we should address this in a different PR, after we have decided our release strategy. 

Also since we are going to be having releases on our own schedule (more frequently than upstream), we cannot follow the upstream versions.

Reply:
Ok, could you create a gh issue or jira task to work on the versioning, so that we keep it in the backlog?